catalog/controller/extension/payment/razorpay.php

```diff
@@ -13,6 +13,24 @@ class ControllerExtensionPaymentRazorpay extends Controller
     const PAYMENT_FAILED        = 'payment.failed';
     const ORDER_PAID            = 'order.paid';
 
+    protected $webhookId = null;
+    protected $webhookUrl = HTTP_SERVER . 'index.php?route=extension/payment/razorpay/webhook';
+    protected $webhookEnable = '1';
+    protected $webhookSecret = null;
+
+    protected $webhookSupportedEvents = [
```

Comment:
This list should have default events list as well.

Reply:
@ramth05 okay